pkg/mcpserver/server.go

```diff
@@ -95,7 +95,9 @@ func runStreamableHttpServer(ctx context.Context, mcpServerConfig *mcpfile.MCPSe
 		}
 
 		return s
-	}, &mcp.StreamableHTTPOptions{})
+	}, &mcp.StreamableHTTPOptions{
+		Stateless: true,
```

Comment:
should it be by stateless by default or just give users an option? and if we choose to set it by default, at least should we allow the mcpfile to override it?

Reply:
Yeah that's a good point - I'll add it to the config

Reply:
I might be misunderstanding, but doesn't this still need to change to be whatever the input config was?

Reply:
oh yeah you sorry, that's my bad - I went and changed the config, but forgot to use it here 🤦